docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.8"
+
+services:
+  db:
+    image: postgres
+    # persist data beyond lifetime of container
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_DB=commcare_sync
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+  redis:
+    image: redis
+    # persistent storage
+    command: redis-server --appendonly yes
+    volumes:
+      - redis_data:/data
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
does this also restart the server when you change the code, like it does outside docker? 
Also, if this process dies because of a syntax error or something, what is the best way of bringing it back up? Running `make start`?

Reply:
Yeah it does. and yeah `make start` would bring it back up.